README.md

```diff
@@ -919,11 +919,11 @@ The output is processed and displayed in the migration dashboard using the in `r
 
 ## [EXPERIMENTAL] Migration Progress Workflow
```

Comment:
It is not experimental anymore if we automate to run it daily

Reply:
I thought about this, but reasoned that we really don't know how well it works yet because I don't think many people (manually) trigger it. As such I thought it prudent to first see what kind of reports come in (if any) before removing the _experimental_ label.

Does that make sense at all?